src/metabase/automagic_dashboards/interesting.clj

```diff
@@ -118,8 +200,7 @@
          (apply math.combo/cartesian-product)
          (map (partial zipmap named-dimensions))
          (map (fn [nm->field]
```

Comment:
This is the only substantial change. The `->reference` changes are just raising the code in the ns to be above this.